### ISSUE TITLE
Property is not defined in schema and the schema does not allow additional properties (Pool)

### ISSUE THREAD

**Issue by Avengerist** [Telmate/terraform-provider-proxmox#995]
Today encountered with the error:
```
│ Error: error updating VM: 400 Parameter verification failed., error status: {"data":null,"errors":{"pool":"property is not defined in schema and the schema does not allow additional properties"}} (params: map[agent:1 bios:seabios boot:order=scsi0 cores:2 cpu:host hotplug:network,disk,usb ide2:none,media=cdrom kvm:true memory:2048 name:Terraform-VM net0:virtio=4A:D1:F5:4A:3A:B0,bridge=vmbr0 numa:false onboot:false pool:vm_pool scsi0:local-lvm:vm-101-disk-0,replicate=0 scsihw:virtio-scsi-single sockets:1 tablet:true vmid:101])
│
│   with proxmox_vm_qemu.terraform,
│   on main.tf line 1, in resource "proxmox_vm_qemu" "terraform":
│    1: resource "proxmox_vm_qemu" "terraform" {
```
Without pool all works good.

**main.tf**:
```tf
resource "proxmox_vm_qemu" "terraform" {
  name         = var.vm_name
  force_create = false
  agent        = 1
  cores        = var.cores
  memory       = var.memory
  boot         = "order=scsi0"
  target_node  = var.node
  pool         = var.pool
  clone        = var.template_name
  full_clone   = true
  scsihw       = "virtio-scsi-single"
  disks {
    scsi {
        scsi0 {
            disk {
                storage = var.storage
                size    = var.size
            }
        }
    }
  }
  network {
    bridge = var.network_bridge
    model  = "virtio"
  }
}
```
**vars.tf:**
```tf
variable "api_url" {
  type    = string
  default = "https://proxmox.local:8006/api2/json"
}

variable "token_id" {
  type    = string
  default = "root@pam!proxmox"
}

variable "token_secret" {
  type    = string
  default = ""
}

variable "node" {
  type    = string
  default = "pve-1"
}

variable "template_name" {
  type    = string
  default = "VM-Template"
}

variable "vm_name" {
  type    = string
  default = "Terraform-VM"
}

variable "pool" {
  type    = string
  default = "vm_pool"
}

variable "network_bridge" {
  type    = string
  default = "vmbr0"
}

variable "storage" {
  type    = string
  default = "local-lvm"
}

variable "cores" {
  type    = number
  default = 2
}

variable "memory" {
  type    = number
  default = 2048
}

variable "size" {
  type    = number
  default = 8
}
```
---
Proxmox Version: **8.2.2**
Terraform Provider Version: **3.0.1-rc1**

**Comment by Tinyblargon** [comment]
This bug has been fixed in the current version of the `master` branch, RC2 will fix this issue.

**Comment by Tinyblargon** [comment]
Fix by v3.0.1-rc2
